.github/scripts/check_symbols.sh

```
@@ -0,0 +1,190 @@
+    error "Library not found at $LIB_PATH"
+    echo "Make sure you have built WasmEdge first with: cmake --build build" 1>&2
+    exit 1
+fi
+
+if [ ! -f "$WHITELIST_FILE" ]; then
+    error "Whitelist file not found at $WHITELIST_FILE"
+    exit 1
+fi
+
+TEMP_DIR=$(mktemp -d)
+trap "rm -rf $TEMP_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)